src/SayMore/UI/ComponentEditors/FieldsValuesGrid.cs

```diff
@@ -344,8 +344,16 @@ protected override void OnCellValuePushed(DataGridViewCellValueEventArgs e)
 			else
 			{
 				string oldId;
+
 				if (!_model.GetShouldAskToRemoveFieldEverywhere(e.RowIndex, e.Value as string, out oldId))
+				{
+					if (e.Value == null)
```

Comment:
I think a comment above this null check is needed. 
Explain why we decrementing the RowCount and returning is needed, and why the value might be null.

Reply:
Ok